Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
"regenerating the types" == generating clientset, conversions, protobuf, etc.

`k8s.io/metrics/pkg/apis/custom_metrics` uses a hendwritten client, `k8s.io/metrics/pkg/apis/metrics` uses an autogenerated client.  Both use autogenerated conversions, etc.  The copy of clientgen used to generate clients for `pkg/apis/metrics` was extracted from the main repo a while ago, and then has a couple of improvements tacked on to handle read-only APIs and such.  Unfortunately, it currently only exists in the form of https://github.com/kubernetes/repo-infra/pull/7, because I haven't had a chance to fix up the rest of the stuff in that PR.

I figured if we moved to staging, I could just port the two tweaks to client-gen over to the main repo's client-gen, and then just use the main repo's code gen scripts to generate conversions, etc (and then sync stuff back to the main k8s.io/metrics repo as per the staging sync process).

In the long run, we'll probably need to make out-of-main-repo codegen easier, but that work will probably come as part of the new https://github.com/kubernetes-incubator/apiserver-builder.